docs/get-details/dapps/avm/teal/opcodes.md

```diff
@@ -1318,7 +1318,7 @@ bitlen interprets arrays as big-endian integers, unlike setbit/getbit
 - Opcode: 0xb3
 - Pops: _None_
 - Pushes: _None_
-- Execute the current inner transaction. Fail if 16 inner transactions have already been executed, or if the transaction itself fails.
+- Execute the current inner transaction. Fail if 256 inner transactions have already been executed, or if the transaction itself fails.
```

Comment:
I know this is true in teal 6, is it also true in teal 5?

Reply:
@barnjamin Based on https://github.com/algorand/go-algorand/blob/3ce0afbdbf3455fd5c1aa8402a3d3529f507187b/config/consensus.go#L1050, I don't think the statement is true in TEAL 5.  

Since the value changes between TEAL versions, what's the preferred recourse?
* Specify limit per TEAL version.
* Specify 256 applies to TEAL >= 6.
* Something else.

Reply:
I think there is a precedent set with something like https://github.com/algorand/go-algorand/blob/master/data/transactions/logic/TEAL_opcodes.md#sha256 maybe? 

Also I think this file happens to be autogen'd from go-algorand so the pr would have to be opened against those docs

Reply:
@barnjamin As sorry, I missed that the docs are auto-generated.

Via https://github.com/algorand/go-algorand/commit/0111cb102c870eb5c107479e13c34cb6d62cb32c#diff-53716e787ca407c6b14a0b83322be4228b6715555616e3ff5e6ca427d03c8810R1228, the go-algorand language is updated to avoid specifying the limit.

Since the change was merged, I'm _assuming_ we're comfortable with the updated language.  Resolving and closing the PR. 

If you'd like to see different changes, feel welcomed to re-open the thread.